src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -193,6 +193,15 @@ public override void Initialize()
             this.logger.LogTrace("(-)");
         }
 
+        /// <summary>
+        /// Prints command-line help.
+        /// </summary>
+        /// <param name="network">The network to extract values from.</param>
+        public static void PrintHelp(Network network)
```

Comment:
Do these have to be static methods?

Reply:
Yes. The features are only being constructed after calling the RunAsync method.